### ISSUE TITLE
req snippet doesn't work for CacheMgr

### ISSUE THREAD

**Issue by foegit** [SqrTT/prophet#347]
req snippet doesn't work for CacheMgr (https://documentation.b2c.commercecloud.salesforce.com/DOC2/topic/com.demandware.dochelp/DWAPI/scriptapi/html/api/class_dw_system_CacheMgr.html)

**To Reproduce**
1. Open any BE script type `reqCache`

**Expected behavior**
The extension suggests a snippet to be inserted `var CacheMgr = require('dw/system/CacheMgr');`

**Screenshots**
![image](https://user-images.githubusercontent.com/32361896/212361876-ffbfc116-2679-4efb-a604-76e6add8251e.png)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - VSCode version: 1.74.2
 - The extension version: 1.4.31

**Comment by SqrTT** [comment]
Hi @foegit 
Thanks for reporting

It will be fixed within in next release.

**Comment by aduponchel** [comment]
You can add the following snippet for Javascript files.

```js
{
	"Require Class Cache": {
        "prefix": "reqCache",
        "body": [
            "const Cache = require('dw/system/Cache');"
        ],
        "description": "Require Class Cache"
    },
	"Require Class CacheMgr": {
        "prefix": "reqCacheMgr",
        "body": [
            "const CacheMgr = require('dw/system/CacheMgr');"
        ],
        "description": "Require Class CacheMgr"
    },
}
```

Please note I use const instead of var.
I noticed before 2019 const was used instead of var.
I don't know why but will appreciate to know :) I always use const except in ISML iscript due to hoisting.

Regards

**Comment by SqrTT** [comment]
Hi @aduponchel 
`const` in not recommend to use since it has non standard behavior in comparison to ES6. For instance it has lexical scope instead of block scope. Try to guess  `result` value after next code:

```javascript
var result = ''

if (true) {
    const test = 'second'
}
result = test
```  
or this code also will work
```javascript
var result = ''
const test
if (true) {
    test = 'second'
}
result = test
```

or this example:
```javascript
const product = getProduct(/*...*/)

if (true) {
    // SFCC will throw error here while ES6 just create new block scoped variable
    const product = getProduct(/*...*/)
}
```
none of this error can be catched by IDE or eslint since tools counts on ES6 standart.

Please note: Salesforce avoid usage of `const` in SFRA (along with `let` and `empty`)

**Comment by SqrTT** [comment]
fixed in 1.4.34
